services/relay.c

```diff
@@ -0,0 +1,66 @@
+#include "jd_services.h"
+#include "interfaces/jd_pins.h"
+#include "jacdac/dist/c/relay.h"
+
+struct srv_state {
+    SRV_COMMON;
+    uint8_t intensity;
+    uint8_t variant;
+    uint32_t max_switching_current;
+    relay_params_t params;
+};
+
+REG_DEFINITION(                                   //
+    relay_regs,                                   //
+    REG_SRV_COMMON,                         //
+    REG_U8(JD_RELAY_REG_CLOSED),                 //
```

Comment:
@mmoskal Either I failed to program the MCU around 20 times or it appeared that the `<tab> <tab> <tab> \\` actually made a difference to compilation. If it is the latter, is there a way I can patch it so that the trailing comment is not required? I was scratching my head for some time.

Reply:
The comments are there only to prevent the clang indent program from collapsing everything into a single line. Either way it's fine. I would be inclined to believe 20 programmings failed rather then the comments make a difference in compilation outcome...